remoteappmanager/cli/remoteapprest/__main__.py

```diff
@@ -137,19 +137,25 @@ def available(ctx):
 
 @app.command()
 @click.argument("identifier")
+@click.option("--startupdata", default=None)
 @click.pass_context
-def start(ctx, identifier):
+def start(ctx, identifier, startupdata):
     """Starts a container for application identified by IDENTIFIER.
+    If ``startupdata`` is provided, the container will open the given file at
+    startup.
```

Comment:
Out of curiosity, what would happen in the situation that the application policy does not allow `startupdata` to be assigned but a file is provided in the CLI?

Presumably at the moment the `start` request should fail - I don't think there's much more we could do since we don't have access to the application database to check this beforehand.

Reply:
Good catch, strangely enough in that case the container starts anyway, with the provided startup data too! I'll see how we can prevent this.